scripts/i18n-messages

```diff
@@ -37,7 +37,23 @@ def main(cmd, args):
     elif cmd == 'add':
         i18n.generate_po(args)
     elif cmd == 'validate':
-        i18n.validate_translations(args)
+        results = i18n.validate_translations(args)
+
+        num_errors = 0
+        for k, v in results.items():
+            if v < 0: # Locale files did not exist
+                print(f'Unknown locale: {k}')
+                sys.exit(1)
```

Comment:
This is a little awkward that the number can represent to different things (very C-y :P ). Let's maybe just not handle the error. We don't need a special error message there if the user specifies an invalid locale.